pkgs/tools/misc/ttyrec/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   patches = [ ./clang-fixes.patch ];
 
-  makeFlags = [ "CFLAGS=-DSVR4" ]
+  makeFlags = stdenv.lib.optional buildPlatform.isLinux "CFLAGS=-DSVR4"
```

Comment:
@therealpxc I am mainly pleased people are figuring out about `*Platform` at all :). This is new to most, so no worries.

[Also, one can now do `stdenv.*Platform.is*`, if you don't want to add an import.